### DISCUSSION TITLE
ISH shell for Android?

### DISCUSSION THREAD

**Discussion by Mizan-Al-Sabbih** [ish-app/ish#1657] [Q&A]
Are you want make the ISH shell for Android?

**Comment by saagarjha** [comment]
There's not really any point, considering that it's far easier to get a Linux-like environment on Android via something like [Termux](https://termux.com).

**Comment by 0x4248** [comment]
> Are you want make the ISH shell for Android?

Would this be impossible since the entire repo is based for apple iOS only.
